### PR TITLE
add tests for missing test cases (throw error)

### PR DESCRIPTION
**Pull request by thiloSchlemmer**

### CODE REVIEW

File: test/injection/core/unregisteredInjection.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { $injector } from './testsConfig';
+var mockService =  { get: 'I\'m a mock. ' };
```
Comment (taulinger) [inline test/injection/core/unregisteredInjection.test.js:2]: 🧐

**Review by taulinger — APPROVED** [review]
well done!

File: test/injection/core/registerTwiceError.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { $injector } from './testsConfig';
+var mockService =  { get: 'I\'m a mock. ' };
```
Comment (taulinger) [inline test/injection/core/registerTwiceError.test.js:2]: 🧐 we don't want vars anymore 😉

Comment (thiloSchlemmer) [reply test/injection/core/registerTwiceError.test.js]: ah yes we make `let`/`const` not `var`... indeed :)

**Pull request merged**